binding/HarfBuzzSharp.Shared/Blob.cs

```diff
@@ -1,35 +1,29 @@
 ﻿using System;
 using System.IO;
-using System.Runtime.InteropServices;
 
 namespace HarfBuzzSharp
 {
 	// public delegates
+	//
+	// bad choices.
+	// this should not have had the "blob" prefix
+	// it is a global dispose method, but we can't switch now
+	// it is a breaking change since it will become ambiguous
 	public delegate void BlobReleaseDelegate (object context);
 
-	// internal proxy delegates
-	[UnmanagedFunctionPointer (CallingConvention.Cdecl)]
-	internal delegate void hb_destroy_func_t (IntPtr context);
-
 	public class Blob : NativeObject
 	{
-		// so the GC doesn't collect the delegate
-		private static readonly hb_destroy_func_t destroy_funcInternal;
-		private static readonly IntPtr destroy_func;
+		private static readonly Lazy<Blob> emptyBlob = new Lazy<Blob> (() => new StaticBlob (HarfBuzzApi.hb_blob_get_empty ()));
```

Comment:
What was the reason for this? The native side should always return the same instance. This is only needed for reference equality on managed side.

Reply:
This WAS for equality - I am using the `Lazy<T>` so it is only ever initialized once - i always return the same instance.

Reply:
We should compare handles instead of .Net instances. HarfBuzz will return the same singleton on every call to get_empty.

Reply:
```
private bool Equals(NativeObject obj)
{
    return Handle == obj.Handle;
}
```